integration.json

```diff
@@ -1,7 +1,6 @@
 {
   "builders": ["paketobuildpacks/builder:buildpackless-base", "paketobuildpacks/builder-jammy-buildpackless-base"],
-  "dotnet-core-aspnet": "github.com/paketo-buildpacks/dotnet-core-aspnet",
```

Comment:
I think we should continue to include integration tests that prove this buildpack still integrates with the dotnet-core-runtime and dotnet-core-aspnet buildpacks, as that's an important acceptance criteria for this implementation.

Reply:
This should be done.